pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractLombokAwareRule.java

```diff
@@ -112,27 +122,15 @@ protected boolean hasLombokAnnotation(Node node) {
         return result;
     }
 
-    protected ASTAnnotation getLombokAnnotation(Node node, String lombokAnnotation) {
```

Comment:
we shouldn't remove this to avoid breaking API changes, but keep it and deprecate it.

Reply:
This method was added to the code source after last release, so we can delete it.

Reply:
True, thanks!